File: API/api.client/src/domain/vehicles/vehiclePhoto.ts
export class VehiclePhoto{
    constructor(
        public id: string,
        public vehicleid: string,
        public path: string
    ) {
    }
}

export function mapToVehiclePhoto(data: any){
    return new VehiclePhoto(data.id, data.vehicleid, data.path)
}

export function mapToVehiclePhotos(data: any[]){
    return data.map(d => mapToVehiclePhoto(d))
}

File: src/RentCars.Domain/Vehicles/VehiclePhoto.cs
namespace RentCars.Domain.Vehicles
{
    public class VehiclePhoto
    {
        public Guid Id { get; }
        public Guid VehicleId { get; }
        public String Path { get; }

        public VehiclePhoto(Guid id, Guid vehicleId, String path)
        {
            Id = id;
            VehicleId = vehicleId;
            Path = path;
        }
    }
}
